flags.go

```diff
@@ -196,10 +196,17 @@ func (f *Flags) setupCommonFlags() {
 		fs.StringVar(&f.Password, "password", f.lookupEnvOrString("AMT_PASSWORD", ""), "AMT password")
 		fs.DurationVar(&f.AMTTimeoutDuration, "t", 2*time.Minute, "AMT timeout - time to wait until AMT is ready (ex. '2m' or '30s')")
 	}
+	for _, fs := range []*flag.FlagSet{
```

Comment:
Instead of two for loops, what about putting a condition in the first for loop?
``` suggestion
if(fs.Name() != "activate"){
		fs.BoolVar(&f.Force, "f", false, "Force even if device is not registered with a server")
}
```

Reply:
done